app/assets/json/translations/react/en.json

```diff
@@ -37,6 +37,14 @@
   "accountSettings.verifyEmail": "Please check your email inbox to verify this change.",
   "alreadySubmitted.message": "You have already submitted an application to this listing.",
   "alreadySubmitted.title": "Good news!",
+  "assistance.contact.helpLine.title1": "For help with San Francisco services 24 hours a day, 7 days a week:",
+  "assistance.contact.helpLine.subtitle": "Call 311 or <a target='_blank' href='https://sf311.org/'>visit the 311 website.</a>",
```

Comment:
should we add a `tel` [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) for 311 to make it easy peasy to dial?

Reply:
Oh duh! Completely forgot about `tel` links. Thank you for catching that!